src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -276,8 +276,10 @@ bool pal::get_default_installation_dir(pal::string_t* recv)
     }
     //  ***************************
 
+    USHORT pProcessMachine, pNativeMachine;
     const pal::char_t* program_files_dir;
-    if (pal::is_running_in_wow64())
```

Comment:
The other place where this is called is in utils.cpp `get_dotnet_root_from_env` - we should fix both places to use the same mechanism.